### PR TITLE
Add Storybook

### PR DESCRIPTION
**Pull request by solomonhawk**
Related #13 

Ran into a whole bunch of small issues along the way, but here we are. 🔥 🔥 🔥  It's fine, everything is fine. 🔥 🔥 🔥 

`+40,106 −11,469` 💪 (`package-lock.json`... sad)

### CODE REVIEW

File: src/global-styles.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  }
+
+  /**
+   * 9. Create a root stacking context
+   */
+  #root,
+  #__next {
+    isolation: isolate;
+  }
+`;
```
Comment (solomonhawk) [inline src/global-styles.tsx:112]: I'm not dead set on this. What ended up being nice here was the ability to import this component in a decorator in storybook and toggle the body background based on the theme.

File: src/global-styles.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  h5,
+  h6 {
+    overflow-wrap: break-word;
+  }
+
+  /**
+   * 9. Create a root stacking context
+   */
+  #root,
+  #__next {
```
Comment (nick-telsan) [inline src/global-styles.tsx:109]: You can yoink out `#__next` since you're just using React.

File: src/global-styles.tsx (hunk 1 above)
Comment (nick-telsan) [reply src/global-styles.tsx:112]: Yeah, this is a bit odd, but it seems reasonable. Emotion is handling your CSS in JS, why not let it handle the rest?

**Review by nick-telsan — APPROVED** [review]
`+40,106 −11,469` What a diff!

**Pull request merged**